skyway_app/views.py

```diff
@@ -5,8 +5,7 @@
 
 
 def index(request):
-  context = {"time_estimate": 5, "GOOGLE_API_KEY": ADD IT HERE}
-  return render(request, 'skyway_app/index.html', context)
+  return render(request, 'skyway_app/index.html')
```

Comment:
The point of this was you would replace the `ADD IT HERE` with the API key. Then the views.py sends the key to the template as a variable